contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,100 @@
+* JavaScript and other client-side scripts, particularly if they are resource-intensive and take a long time to execute.
+* Caching, since it can reduce the time it takes to download and render resources.
+*  Network Latency
+
+## Dom Interactive
+[Dom Interactive](https://developer.mozilla.org/en-US/docs/Web/API/Document/readyState) means that all HTML content on the page has been downloaded and parsed and is ready to be manipulated by JavaScript code. However, sub-resources such as scripts, images, stylesheets and frames are still loading. In general, DOM Interactive happens before FCP (since FCP requires the browser to have downloaded and rendered at least some content of the page). Any subsequent changes to the DOM beyond the inital page load also trigger DOM Interactive and thus it can be called multiple times.
+
+Factors affecting DOM Interactive are:
+* Page Size and Complexity. The larger and more complex a page is, the longer it takes for the browser to parse and process the HTML content, leading to a slower DOM Interactive time.
+* Number and Size of Scripts. The more scripts a page has and the larger their size, the longer it takes for the browser to download and execute them, which can delay DOM Interactive.
+* Server Response Time: Slow server response times can delay the delivery of the page's resources, including the HTML content, leading to slower DOM Interactive times.
+* Third-party Scripts.
+* Network Latency.
+
+## Page Loaded
+The whole page has [loaded](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) and is ready for the user to interact with it, including all dependent resources such as stylesheets and images. 
+
+Note that this is not the same as [Time to Interactive](https://web.dev/tti/) (TTI), which is a more comprehensive metric. When the Page Loaded event is trigged, some page elements may still be loading or processing, and the page may not be fully responsive to user interactions. This is in contrast to TTI, which measures how long it takes to be able to interact with all the page's features and functionality (although Page Loaded is a big contributing factor to TTI).
+
+For example, imagine you are filling out a complex form on a website with many input fields, but the form is slow to respond when you try to interact with it. Even after the page has finished loading, the validation scripts that check if you've entered the correct information might still be running in the background, causing the form to be slow or unresponsive. TTI measures how long it takes for the page to become fully interactive and responsive to user inputs.
+
+While there is no benchmark for what is a good time for Page Loaded, a [good time for TTI](https://developer.chrome.com/docs/lighthouse/performance/interactive/#how-lighthouse-determines-your-tti-score) is under 3.8 seconds while over 7.3 seconds is considered slow.
+
+
+## So what's more important? FCP, Dom Interactive, or Page Loaded?
```

Comment:
For this section, we generally try to avoid saying "it depends" which you've done a nice job with. 

It would be nice if the bullet points were all types of sites or apps. What would a B2B SaaS, blog, social network, B2C startup care about? When would you not care as much about these?

I think you can fill this section out more.

Reply:
> It would be nice if the bullet points were all types of sites or apps. What would a B2B SaaS, blog, social network, B2C startup care about? When would you not care as much about these?

Nice idea!